examples/service-clients/azure-client/external-controller/tests/index.ts

```diff
@@ -5,7 +5,7 @@
 
 /* eslint-disable import/no-internal-modules */
 
-import { type ISharedMap, SharedMap } from "@fluidframework/map/internal";
+import { SharedMap } from "@fluidframework/map/internal";
```

Comment:
Ideally, examples should not use `/internal` imports. This should probably be
```suggestion
import { SharedMap } from "fluid-framework/legacy";
```

Reply:
(We should also remove direct DDS package dependencies)

Reply:
Yep - don't know the replacements. :/